utils/smtp_client.py

```diff
@@ -27,8 +30,14 @@ def init(host, port, username, password):
         s.starttls()
         s.login(username, password)
         _client = s
+    if _server is None:
```

Comment:
Does it make sense to init this every time if it's only going to be used by `get_sentry_users_from_mails`? Is there a pythonic way to do lazy inits? :)

Reply:
Most likely we would need to turn this into an object to achieve this kind of behaviour